src/components/Dropdown/DropdownComponentHover.vue

```diff
@@ -0,0 +1,90 @@
+<template>
```

Comment:
I think we can have a single component called Dropdown with a `trigger` prop which takes a string, either `"hover"` or `"click"` and change the `isVisible` ref accordingly. What do you think? We wouldn't need 2 different components for this.

Reply:
I agree with Yash. I didn't do a deep review as my first impression was dropdown not closing on clicking outside. @YashKandalkar what do you think, will it be good to have?